rcl/src/rcl/node_options.c

```diff
@@ -36,7 +36,6 @@ rcl_node_get_default_options()
     .arguments = rcl_get_zero_initialized_arguments(),
     .enable_rosout = true,
     .rosout_qos = rcl_qos_profile_rosout_default,
-    .enable_type_description_service = false,
```

Comment:
Hm.  I kind of feel like we should assign `enable_type_description` in this backport.  I know it is unused, but it feels wrong to let it be an arbitrary value, rather than hard-coding it to `false`.  The same goes during the copy below.  What do you think?

Reply:
Yes, i agree. Missed that in the backport edit